draft-meynell-panrg-scion-research-questions.md

```diff
@@ -200,6 +200,26 @@ This draft assumes the reader is familiar with some of the fundamental concepts
 * One AS belongs to many ISDs?
 
 
+## CORE Beaconing Scalability
+
+The idea behind beaconing is to discover all possible paths (loop free and with a fixed maximu length) between two CORE AS.
+Every AS forwards any received becons to all neighbor CORE ASes unless this would cause a loop or exceed the fixed maximum length.
+Implemented naively, the number of paths (and beacons) grows superexponentially with the network size.
```

Comment:
Two things: 
- I'd clarify the context for this number.. Is it in total in an overall SCION network? Is it per AS? I think it makes sense to talk about numbers per AS in general
- According to our (@matzf ) calculations and my understanding, if you limit the number of beacons with the "best PCB" practice, then the number is o(N*log(N)). So this would be if you were to not filter the number of beacons at all?